IdentityCore/src/auth_scheme/MSIDAuthenticationSchemePop.m

```diff
@@ -64,14 +66,23 @@ - (instancetype)initWithSchemeParameters:(NSDictionary *)schemeParameters
         }
         
         NSString *kidJwk = [_req_cnf msidBase64UrlDecode];
+        if (!kidJwk)
+        {
+            MSID_LOG_WITH_CTX(MSIDLogLevelError, nil, @"Failed to read req_cnf");
+            return nil;
+        }
         NSData *kidData = [kidJwk dataUsingEncoding:NSUTF8StringEncoding];
-        
+        if (!kidData)
```

Comment:
Can we check !kidData && [kidData length] != 0 just to be safe

Reply:
Addressed. Will be in next update